.devcontainer/nb-to-package/devcontainer.json

```diff
@@ -0,0 +1,13 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+{
+  "name": "Notebooks to Packages",
+  "image": "ghcr.io/uw-ssec/minimal:latest",
```

Comment:
What's in this? Can we instead just point to the current CryoCloud image? (Or what packages are not in there that you'd need for this tutorial) https://github.com/CryoInTheCloud/hub-image

Reply:
This image is lean and has `pixi` installed. This tutorial doesn't really need anything fancy :)